data/download.yaml

```diff
@@ -0,0 +1,42 @@
+
+binaries:
+  platforms:
+  - darwin
+  - linux
+  - windows
+  platformInfo:
+    darwin:
+      name: macOS
+      icon: fa-apple
+    linux:
+      name: Linux
+      icon: fa-linux
+    windows:
+      name: Windows
+      icon: fa-windows
+  fullReleaseVersions:
+  - "1.4.1"
+  - "1.5.0"
+  - "1.6.0"
+  partialReleaseVersions:
+  - "1.4.0"
+  - "1.3.0"
+  - "1.2.0"
+  - "1.1.0"
+  - "1.0.0"
+  - "0.9.0"
+  - "0.8.0"
+  - "0.7.0"
+  - "0.6.0"
+  - "0.5.2"
+  - "0.5.1"
+  - "0.5.0"
+docker:
+- name: jaeger-agent
+  description: The Jaeger agent
+- name: all-in-one
+  description: The "all-in-one" hot rod example used in the [getting started](/docs/latest/getting-started) guide
```

Comment:
We explain all-in-one in Getting Started as "designed for quick local testing, launches the Jaeger UI, collector, query, and agent, with an in memory storage component."  I don't think it makes sense to refer to the HotROD in the description, as the purpose is just having a single binary to run full Jaeger.